Pj_MonteVerdi_defender.gmx/scripts/on_update.gml

```diff
@@ -16,13 +16,39 @@ if( vertical_speed < max_vertical_speed ) {
     vertical_speed += gravity_acceleration;
 }
 
+//Spawn platform
+if( platform_spawn ) {
+    platform_vertical_separation = 4;
+    new_platform = instance_create(x, y+sprite_height/2+platform_vertical_separation, obj_platform_Orpheus) 
+    platform_spawn = false;
+}
+if( sustain_platform ) {
+    with( new_platform ) {
+    time--;
+    }
+    if( keyboard_check_released( vk_space ) ) {
+        sustain_platform = false;
+    }
+}
 //update speed on collision
-collision = find_collision( obj_soil );
+collision_object = obj_city_ground;
+find_collision( collision_object );
```

Comment:
I think that fin_collision() should return a value instead of updating a variable